ensure_command_output.sh

```diff
@@ -1,59 +1,43 @@
 #!/bin/bash
 
-MAGIC_END_STRING="END OF SUBCOMMAND"
 OUTPUT_SLEEP=60
 
 function run_command {
-  PIPE=$1
-  shift
   CMD="$@"
-  $CMD 1>$PIPE 2>$PIPE
-  echo "$MAGIC_END_STRING" 1>$PIPE
+  $CMD 
 }
 
 function run_ping {
-  PIPE=$1
-  shift
   CMD="$@"
-  (
     while [ 1 ]
     do
       echo "Command '$CMD' is still running."
       sleep $OUTPUT_SLEEP
     done
-  ) >$PIPE
 }
 
 function presto_wait {
```

Comment:
What this method has in common with presto? Maybe it is better to express the context of this method within it's name?